src/plugin.js

```diff
@@ -29,13 +29,11 @@ class Plugin {
   }
 
   substitutePaths(jsonld) {
-    jsonld = jsonld.slice()
-    for (let items of jsonld) {
-      for (let item of items['@graph']) {
-        if (!item.photo) continue
-        for (let photo of item.photo) {
-          photo.path = this.destination(photo.path, '.')
-        }
+    jsonld = Object.assign({}, jsonld)
```

Comment:
You can skip this, I think? I assume the idea was to make a defensive copy, but this is only a shallow copy, as was the original `.slice()` call.

Reply:
Skipped.